### PR TITLE
Fix tx review screen and improve them

### PR DESCRIPTION
**Pull request by xchapron-ledger**

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -352,7 +352,7 @@ unsigned char io_event(unsigned char channel) {
 			  SEPROXYHAL_TAG_STATUS_EVENT_FLAG_USB_POWERED)) {
 			THROW(EXCEPTION_IO_RESET);
 		}
-		/* fallthrough */
+		__attribute__((fallthrough));
```
Comment (sgliner-ledger) [inline src/main.c:355]: nit: typo in commit name

**Comment by xchapron-ledger** [comment]
Clang checks fails due to warning that were previously there.

File: tests/functional/test_version_cmd.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # Taken from the Makefile, to update every time the Makefile version is bumped
 MAJOR = 0
 MINOR = 5
-PATCH = 1
+PATCH = 2
```
Comment (sgliner-ledger) [inline tests/functional/test_version_cmd.py:6]: Can't you get this info from OCR ?

**Review by sgliner-ledger — APPROVED** [review]

File: tests/functional/test_version_cmd.py (hunk 2 above)
Comment (xchapron-ledger) [reply tests/functional/test_version_cmd.py:6]: It is checked also on the screen in another test. Here we are comparing the response to the get version apdu.

File: src/main.c (hunk 1 above)
Comment (xchapron-ledger) [reply src/main.c:355]: fixed

**Pull request merged**